packages/drivers/src/servers.ts

```diff
@@ -0,0 +1,66 @@
+import * as ds from "@devicescript/core"
+import { SensorServer, startServer } from "@devicescript/server"
+
+export interface TemperatureOptions {
+    min: number
+    max: number
+    error: number
+    read: () => Promise<number>
+}
+
+export class TemperatureServer
+    extends SensorServer<ds.ITemperatureServer>
+    implements ds.ITemperatureServer
+{
+    constructor(public options: TemperatureOptions) {
+        super(ds.Temperature.spec, "temperature")
+    }
+
+    async temperature() {
+        return await this.options.read()
+    }
+    minTemperature() {
```

Comment:
min -> Temperature
temperature -> Error

should we always order things with name + suffix?

Reply:
I don't understand; the order and names come from the spec